packages/titus-db-manager/Dockerfile

```diff
@@ -1,19 +1,14 @@
-FROM node:16-alpine
-RUN apk add bash dumb-init
+FROM node:16.13.1-alpine3.14
 ENV NODE_ENV production
-
 WORKDIR /titus-db-manager
-
-COPY --chown=node:node index.js /titus-db-manager/index.js
-COPY --chown=node:node migration-start.js /titus-db-manager/migration-start.js
-COPY --chown=node:node seed /titus-db-manager/seed
-COPY --chown=node:node migrate /titus-db-manager/migrate
-COPY --chown=node:node truncate /titus-db-manager/truncate
-COPY --chown=node:node package.json package-lock.json ./
-COPY --chown=node:node lib /titus-db-manager/lib
-
-RUN npm ci --only=production
-
+COPY --chown=node:node package.json package-lock.json index.js migration-start.js ./
+COPY --chown=node:node lib lib
+COPY --chown=node:node seed seed
+COPY --chown=node:node migrate migrate
+COPY --chown=node:node truncate truncate
```

Comment:
is there a smarter way to do this? it feels overspecified (I do realize it was already like this before)

Reply:
It's the expected behaviour of the COPY command when working with directories (this doesn't happen with just files, as you can see in the first line). Trying to reduce the COPY layers lead us inevitably to some kind of workaround, like the one I describe below.

If you want to copy multiple directories (not their contents) under a destination directory in a single command, you'll need to set up the build context so that your source directories are under a common parent and then COPY that parent. So we'd have to move the three folders under one parent folder and add the parent. That may have a huge impact in the source code or even be impossible to do.

Other dirty hacks would go through "tarring" the directories and COPYing the tar file, using several FROM directives or (ugh) playing with the Unix `cp` command. As you can see, overengineering...

Reply:
what about using dockerignore files?

Reply:
I wouldn't use a static list of files to be ignored from a massive COPY command. If a dev adds files/folders or modifies the repo filesystem, we could end with unwanted files/folders in the image.

Reply:
fair enough